rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -98,17 +100,74 @@ def __str__(self) -> str:
         )
 
 
-@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class BaseAcquisitionsOrder(Iterator, metaclass=ABCMeta):
```

Comment:
I think files and structures in the accounting module could be rearranged because I've tried to put orders-related classes into a separate file but couldn't due to cyclic import

Reply:
So what is the best right now: moving the orders logic to a separate module or leaving it as is?